File: FFF_Elis/EPAS.BusinessLogic/Dtos/OperationResponseDto.cs
using EPAS.Core.Enums;
using EPAS.Core.Models;

namespace EPAS.BusinessLogic.Dtos;

public class OperationResponseDto
{
    public string OperationId { get; set; }
    public string Username { get; set; }
    public OperationResponseEnum Response { get; set; }
    public List<Qualifications> Qualifications { get; set; } = new();
}

File: FFF_Elis/EPAS.App/epas_app/lib/models/Dtos/OperationResponseDto.dart
import 'package:epas_app/models/Enums/OperationResponseEnum.dart';
import 'package:epas_app/models/Qualifications.dart';

class OperationResponseDto {
  String operationId;
  String username;
  OperationResponseEnum response;
  List<Qualification> Qualifications;

  OperationResponseDto(
      this.operationId, this.username, this.response, this.Qualifications);
}
